json-rpc/src/runtime.rs

```diff
@@ -70,7 +70,7 @@ struct RpcResponseLog<'a> {
 #[derive(serde::Deserialize)]
 struct HealthCheckParams {
     // Health check returns 200 when this param is provided and meet the following condition:
-    //   server latest ledger info timestamp >= server current time timestamp + duration_secs
```

Comment:
is this a change for exist typo or on purpose?

Reply:
This is fixing a typo, since the code actually does:

timestamp >= server current time timestamp - duration_secs